bindings/python/runtime.cpp

```diff
@@ -6,6 +6,6 @@ void bind_runtime(py::module_ &m) {
   auto runtime = m.def_submodule("runtime", "K LLVM backend runtime");
 }
 
-PYBIND11_MODULE(kllvm_static, m) {
+PYBIND11_MODULE(kllvm_runtime, m) {
```

Comment:
Is there a way to include the kompiled definiton's name in the module name?

Reply:
No, unfortunately not without making pretty radical changes to the entire build process for the bindings. This module name needs to be set statically here so that the generated code knows what init function to call when the shared library is loaded by Python.

I'd view this as being similar to `llvm-kompile` generating an executable called `interpreter`; the files are identified by the kompiled directory they live in, rather than their own names.

Reply:
I see, makes sense.